test/bin/common.sh

```diff
@@ -87,6 +87,20 @@ SCENARIO_INFO_DIR="${SCENARIO_INFO_DIR:-${IMAGEDIR}/scenario-info}"
 # The CI system will override this depending on the job its running.
 SCENARIO_SOURCES="${SCENARIO_SOURCES:-${TESTDIR}/scenarios}"
 
+# Directory where all the scenarios will be copied for execution
+#
+# shellcheck disable=SC2034  # used elsewhere
+SCENARIOS_TO_RUN="${OUTPUTDIR}/scenarios-to-run"
+
+prepare_scenario_sources() {
```

Comment:
Nit: can we add docs to this function to explain why we're doing this?

Reply:
`scenarios-periodics/el92-crel@upgrade-fails.sh` includes an example of a scenario skipping itself. Instead of manipulating the scenario files that are inputs to the job, can we do something similar with this one? For example, can we have it check the `CI_JOB_NAME`?

Reply:
@ggiguash Added docs.

@dhellmann This one is a bit different. The upgrade test checks whether it should run based on the ostree commits. This test needs to be skipped under specific CI jobs, not based on anything that you could look up on the VM/host. I chose to do it in these scripts because these are the ones that CI is concerned about. The test is decoupled from how/where it is called, thats a higher level responsibility that ci_*.sh scripts take care of.
Also, the manipulation of tests files that need to run also happens within a CI context.